scripts/dnd5e-helpers.js

```diff
@@ -778,7 +816,9 @@ Hooks.on("preUpdateActor", async (actor, update, options, userId) => {
   if ((game.settings.get('dnd5e-helpers', 'gwEnable')) && (hp !== undefined)) {
     GreatWound_preUpdateActor(actor, update);
   }
-
+if((game.settings.get('dnd5e-helpers', 'owHp0')) && (hp === 0)){
+  OpenWounds(actor)
```

Comment:
function OpenWounds() doesn't take an argument

Reply:
I may have not finished OpenWounds by the look of things 👍